gateware/cas.py

```diff
@@ -0,0 +1,66 @@
+"""
+Module which allows control via buttons and switches and status reporting via
+LEDs.
+"""
+
+from mibuild.generic_platform import ConstraintError
+
+from migen.bank.description import AutoCSR
+from migen.fhdl.std import *
+from migen.genlib.misc import WaitTimer
```

Comment:
We don't seem to be using it yet (the WaitTimer module). Probably you are planning to use it for 5ms delay before setting CSR, in the TODO?